test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -43,4 +44,29 @@ private void assertAreUnique(Object... objects) {
     private void assertNotUnique(Object... objects) {
         assertFalse(Utils.elementsAreUnique(Arrays.asList(objects)));
     }
+
+    @Test
+    public void isAnyNullTest() throws Exception {
```

Comment:
Don't have to append `Test` to tests,
Test method names should be in 3 parts: 
`FeatureUnderTest_scenarioBeingTested_expectedOutcome()`
E.g., `isAnyNull_nullList_returnsTrue()`
Third part or both second and third parts can be omitted depending on what’s covered in the test.
If you are testing both true and false case, OK to remove `returnsTrue`
If you are testing all scenarios, OK to remove `nullList`

In your case, `isAnyNull` works since you are testing all scenarios